src/Sulu/Bundle/MediaBundle/Media/Storage/StorageInterface.php

```diff
@@ -29,6 +32,16 @@ public function save(string $tempPath, string $fileName, array $storageOption =
      */
     public function load(array $storageOption);
 
+    /**
+     * Returns the path for the given file.
+     */
+    public function getPath(array $storageOption): string;
+
+    /**
+     * Returns the path-type for the given file.
+     */
+    public function getType(array $storageOption): string;
```

Comment:
`getPathType` ???

Reply:
yes :)

Reply:
not sure if it is really related to the path as said it could be from `type` => `stream` where you then use `$storage->load()` to create a stream response instead of `getPath`

Reply:
sounds familiar :) @danrot what do you think?

Reply:
`getType` ACK